### DISCUSSION TITLE
How do you plan to make Citadel better/different from Calibre?

### DISCUSSION THREAD

**Discussion by ovizii** [every-day-things/citadel#10] [Q&A]
I did read the project goals and all I can extract from there is:
- be better looking (UX)
- be faster, whatever that applies to

Any more pain points with Calibre you want to share as I assume those are what you want to better with Citadel.

**Comment by phildenhoff** [comment]
I'd be interested to hear what your expectations for a replacement are! I'm primarily writing Citadel to solve my own problems, but I'd love for it to solve yours as well.

**Reply by ovizii** [reply]
I don't have man y problems as I am a pretty new user of Calibre. I'm pretty sure 99% of my problems will be solved as I become more used to working with Calibre. 

The only thing I really dislike atm is the fact that it is a desktop app and I have to run some dockerized version of it, and I am accessing it via a browser over VNC. Would be nice to have an alternative that is a native web application. Not sure what is the case with citadel though.

**Comment by Jawzper** [comment]
Just discovered this project... I for one would really like the option to *not restructure my library's subfolder system.* This is my dealbreaker gripe with Calibre - it insists that I either relinquish control of how my library is organized, or just live with the fact that I have hundreds of gigabytes of duplicates now. This is unnecessary and user-hostile, and I'm sure I'm not the only person who has been traumatized by a music library application similarly deciding that it knows better where to put all my tracks.

Unfortunately the Calibre devs have taken a firm "will not fix" stance on this, so I'm begging you to be kinder to your users and please provide the option.

Personally, I don't care if Calibre's file system is better and more efficient once you get used to it. I simply don't have any plans to work on my library enough to make any kind of file restructuring practical. My library is quite large and complex, my existing folder structure has a purpose, and I don't want to spend hours translating that complexity and purpose into tags and metadata just because the Calibre dev thinks I'm being inefficient. The existing structure is "good enough" for me, and that should be good enough for Calibre.

**Reply by phildenhoff** [reply]
Very reasonable! Can you share how your library is organized?

**Reply by Jawzper** [reply]
It's nothing fancy, I'm not inventing a new Dewey decimal system or anything. I just have folders like manga, comics, TTRPG references, game manuals, fiction, nonfiction, and research articles, further subdivided into categories that are useful to me, ie. by subject matter or series. File names are all over the place but at least I know where to find things.

Objectively, this is a worse system than what Calibre offers. And that's fine.

**Reply by tscahill** [reply]
My ebook organization structure is informed by a few high-school and university newspaper gigs. To wit: Who? What? Where? When? Why? How?

informed, but not strictly modelled on W5H, that would require a multi-dimensional db to handle the storage, and save me the agony of deciding which single dimension a complex book should be filed by. 

Owing much to the just referenced existential crisis, my organizational structure is better than Chaos, but due to the strict linear hierarchy of the structure, can never do the collection justice. I'm appending a text file listing directory structure to the 4th level.  

I'm just coming to your project today. Gonna give it a go.

>tree -d -L 4 > tree.txt
[tree.txt](https://github.com/user-attachments/files/26544104/tree.txt)
